reflect_test.go

```diff
@@ -109,3 +109,79 @@ func TestSchemaGeneration(t *testing.T) {
 		})
 	}
 }
+
+// TestUserOneOf struct
```

Comment:
This comment is useless. If linter is complaining, could we just make struct private?

Reply:
removed.